src/domains/ecommerce-app/api/ecommerce-checkout/v1/_openapi.json.tpl

```diff
@@ -1792,6 +1792,10 @@
             "items": {
               "$ref": "#/components/schemas/Range"
             }
+          },
+          "isOnboardable": {
```

Comment:
I would made this field mandatory in payment method response

Reply:
And if not present on the collection we set it to false by default ?

Reply:
Yes, I would handle the field absence into payment method logic (or make isOnboardable field mandatory for all payment method documents)

Reply:
Fixed with 09f90ad8f46c088662fd63be94ccea3e85947e3f